src/home/PreviewCard/PreviewCard.jsx

```diff
@@ -1,13 +1,42 @@
-import React from 'react';
-import './PreviewCard.css';
+import React from "react";
+import Button from "@mui/material/Button";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardActions from "@mui/material/CardActions";
+import Typography from "@mui/material/Typography";
+import Box from "@mui/material/Box";
 
-export const PreviewCard = ({
-    author,
-    prompt,
-    title,
-}) => <div className="card">
-    <h1>{title}</h1>
-    <h2>{author}</h2>
-    <p className="truncate">{prompt}</p>
-    <button>Challenge</button>
-</div>;
+export const PreviewCard = ({ author, prompt, title }) => (
+  <Card outlined sx={{ width: "18rem", height: "14rem", padding: "2rem" }}>
+    <Box
+      sx={{
+        display: "flex",
+        flexDirection: "column",
+        justifyContent: "space-between",
+        height: "100%",
+      }}
+    >
+      <CardContent
+        sx={{
+          height: "50%",
```

Comment:
do they have a better centering mechanic here instead of just cssing it? I do this all the time too so small nit